go.mod

```diff
@@ -18,8 +18,8 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/ryanuber/go-glob v1.0.0
 	github.com/stretchr/testify v1.6.1
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
-	google.golang.org/api v0.29.0
+	golang.org/x/oauth2 v0.0.0-20210113205817-d3ed898aa8a3
```

Comment:
Unfortunately, this change introduces an incompatibility of dependencies in Vault. I opened a PR in cap (https://github.com/hashicorp/cap/pull/26) to set this version back. Happy to discuss alternatives!

Reply:
I merged https://github.com/hashicorp/cap/pull/26, so this PR should be good to go once you vendor the latest from cap again.